softlayer/cloud-config.yml

```diff
@@ -0,0 +1,51 @@
+azs:
+- name: z1
+  cloud_properties:
+    datacenter: { name: ((sl_datacenter)) }
+
+vm_types:
+- name: default
+  cloud_properties:
+    bosh_ip: ((internal_ip))
+    startCpus:  4
+    maxMemory:  8192
+    ephemeralDiskSize: 100
+    hourlyBillingFlag: true
+    vmNamePrefix: ((sl_vm_name_prefix))
+    domain: ((sl_vm_domain))
+    datacenter:
```

Comment:
OK, I will have test to make sure our CPI works after removing `datacenter` from `vm_types`